concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -563,7 +563,7 @@ Hale-1973-1798-561	Time_Weather_Geography-7	day	n	05a.07	1225	DAY (NOT NIGHT)
 Hale-1973-1798-562	Time_Weather_Geography-8	dawn	n	05a.08	729	DAWN
 Hale-1973-1798-563	Time_Weather_Geography-9	morning	n	05a.09	1339	MORNING
 Hale-1973-1798-564	Time_Weather_Geography-10	afternoon	n	05a.10	1807	AFTERNOON
-Hale-1973-1798-565	Time_Weather_Geography-11	twilight	n	05a.11		
+Hale-1973-1798-565	Time_Weather_Geography-11	twilight	n	05a.11	2297	DUSK
```

Comment:
This should be correct if we consider that WordNet lists "dusk" and "twilight" as synonyms. http://wordnetweb.princeton.edu/perl/webwn?s=dusk&sub=Search+WordNet&o2=&o0=1&o8=1&o1=1&o7=&o5=&o9=&o6=&o3=&o4=&h=

Reply:
Also, we have other _twilight_ glossed mapped to DUSK.